keyboards/squiggle/info.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
Could you please also move this to `rev1/` and add entries for the `LAYOUT_split_3x5_3` and `LAYOUT_reversed`? I think that's the last thing to be done here 👍

Reply:
to confirm. i have to move info.json to `rev1`, correct?
and by entries do you mean new keymaps like `default`, `defaultminidox`, and `rick`?

Reply:
You ahve three layout macros defined in rev1.h, but only one of them is present in the info.json. The Configurator needs them to display them correctly, otherwise they'll all be 1u keys with no spacing.